src/filterPage/ButtonList.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import OptionButton from './OptionButton'
+import { optionList } from '../app/App'
+import { NavLink } from 'react-router-dom'
+
+const StyledOptionButtonList = styled.section`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  padding: 20px;
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
```

Comment:
As grid-column-gap and grid-row-gap have the same value, you could replace these two lines with grid-gap: 20px.